### DISCUSSION TITLE
Norwegian Translation

### DISCUSSION THREAD

**Discussion by kongjudas** [dolezsa/thermal_comfort#51] [Ideas]
I'm too noob to understand this GitHub thingy, but here's the Norwegian translation:

{
  "state": {
    "thermal_comfort__thermal_perception": {
      "dry": "Litt tørt",
      "very_comfortable": "Veldig komfortabelt",
      "comfortable": "Komfortabelt",
      "ok_but_humid": "Greit, men fuktig",
      "somewhat_uncomfortable": "Litt ukomfortabelt",
      "quite_uncomfortable": "Veldig fuktig, ganske ukomfortabelt",
      "extremely_uncomfortable": "Ekstremt ukomfortabelt, trykkende",
      "severely_high": "Skadelig, dødelig for astmastikere"
    }
  }
}

**Comment by rautesamtr** [comment]
Thank you! Added. You should see your translation after the next release.

**Reply by hughc-hub** [reply]
May be is a weird question but I couldn´t find the answer.. how or where should I set the language I want to use with it? thank you!
